### PR TITLE
fix(content-page): load new content page if route changes

### PR DESCRIPTION
**Pull request by bertyhell**
closes: https://district01.atlassian.net/browse/AVO2-952

### CODE REVIEW

File: src/content-page/views/ContentPage.tsx [hunk 1]
```diff
@@ -18,8 +18,16 @@ interface ContentPageDetailProps extends DefaultSecureRouteProps {}
 const ContentPage: FunctionComponent<ContentPageDetailProps> = ({ match, user }) => {
 	const [t] = useTranslation();
 
+	const getCurrentPath = () => `/${(match.params as any).path}`;
```
Comment (andrych) [inline src/content-page/views/ContentPage.tsx]: Hierboven `interface ContentPageDetailProps extends DefaultSecureRouteProps<{ path: string }> {}` dan mag die `as any` weg.

Comment (andrych) [reply src/content-page/views/ContentPage.tsx]: Also: `const getCurrentPath = useCallback(() => `/${match.params.path}`, [match.params.path]);` om onderstaande warning op te lossen.

File: src/content-page/views/ContentPage.tsx [hunk 2]
```diff
@@ -18,8 +18,16 @@ interface ContentPageDetailProps extends DefaultSecureRouteProps {}
...
+
 	// State
-	const [path] = useState<string>(`/${(match.params as any)['path']}`);
+	const [path, setPath] = useState<string>(getCurrentPath());
+
+	useEffect(() => {
+		if (path !== getCurrentPath()) {
+			setPath(getCurrentPath());
+		}
+	}, [match]);
```
Comment (andrych) [inline src/content-page/views/ContentPage.tsx]: Warning: React Hook useEffect has missing dependencies: 'getCurrentPath' and 'path'. `match` mag weg.

File: src/content-page/views/ContentPage.tsx (hunk 1 above)
Comment (bertyhell) [reply src/content-page/views/ContentPage.tsx]: ok, overal gefixed

**Pull request merged**